github/ipld.yml

```diff
@@ -209,12 +209,13 @@ repositories:
     visibility: public
   go-car-priv:
     collaborators:
+      admin:
+        - Jorropo
```

Comment:
Why ? I don't understand how the bot made this choice @galargh

Reply:
This PR was created by a workflow which checks what's the actual GitHub state (it does this by hitting GitHub API directy) and updates both TF state and the YAML config file accordingly.

In this particular case, when we make the call to GitHub asking for your role in `go-car-priv`, it responds with `admin` now because you're a member of `w3dt-stewards` team which has `admin` permission to that repository.

In UI it looks like that:
<img width="766" alt="Screenshot 2022-08-29 at 10 44 55" src="https://user-images.githubusercontent.com/6688074/187162201-efb9c4f6-8d09-4818-baaa-659fa6146bdc.png">

I'd argue that this collaborator relationship is not needed anymore at all - https://github.com/ipld/github-mgmt/pull/19